mongodb-log.yml.example

```diff
@@ -0,0 +1,7 @@
+logs:
+# This sample file will forward mongodb error logs to NR once it's renamed to mongodb-log.yml
```

Comment:
could we follow the same style for the comment of https://github.com/newrelic/nri-redis/pull/134/files?

Reply:
Done.